vpr/src/place/place.cpp

```diff
@@ -1356,16 +1357,7 @@ static float starting_t(const t_annealing_state* state, t_placer_costs* costs, t
     VTR_LOG("std_dev: %g, average cost: %g, starting temp: %g\n", std_dev, av, 20. * std_dev);
 #endif
 
-    float init_temp = 0.0;
-
-    /* We use a constructive initial placement and a low starting temperature 
-     * by default, but that can cause problems with NoCs as the initial logical
-     * locations are random. Use a higher starting T in that case.*/
-    if (noc_opts.noc) {
-        init_temp = 20. * std_dev;
-    } else {
-        init_temp = std_dev / 64;
-    }
+    float init_temp = std_dev / 64;
```

Comment:
Probably good to put a comment in saying this init_t is chosen to not totally destroy the initial placement (which is better than random).